VERSION

```diff
@@ -1 +1 @@
-1.2.0
+1.3.0-prerelease
```

Comment:
I believe the rest of Elastic is using `-SNAPSHOT` for this purpose. I wonder if we should do the same even though this is not a Java project.